### PR TITLE
Release/1.10.0

### PR DESCRIPTION
**Pull request by wshands**

### CODE REVIEW

**Review by denis-yuen — APPROVED** [review]

**Review by denis-yuen — CHANGES_REQUESTED** [review]
Think maybe the merge commit solving the conflicts was not uploaded

File: dockstore-cli-integration-testing/pom.xml [hunk 1]
```diff
@@ -45,7 +45,7 @@
         <dependency>
             <groupId>io.dockstore</groupId>
             <artifactId>dockstore-client</artifactId>
-            <version>1.9.1-SNAPSHOT</version>
+            <version> 1.11.0-alpha.0-SNAPSHOT</version>
```
Comment (denis-yuen) [inline dockstore-cli-integration-testing/pom.xml:48]: Looks like an extra space in here and above

File: support/pom.xml [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>support</artifactId>
+    <packaging>jar</packaging>
+    <name>support</name>
+
+    <parent>
+        <version> 1.11.0-alpha.0-SNAPSHOT</version>
```
Comment (denis-yuen) [inline support/pom.xml:25]: Think the extra space is across the board

**Review by denis-yuen — APPROVED** [review]
After potential version fix

File: dockstore-cli-integration-testing/pom.xml (hunk 1 above)
Comment (wshands) [reply dockstore-cli-integration-testing/pom.xml]: fixed

File: support/pom.xml (hunk 2 above)
Comment (wshands) [reply support/pom.xml]: fixed

**Comment by coverbeck** [comment]
> This pull request **introduces 2 alerts** when merging [f557469](https://github.com/dockstore/dockstore-cli/commit/f5574693e9b70eee90414a80e14e1739c1ca4cb8) into [b3dfa67](https://github.com/dockstore/dockstore-cli/commit/b3dfa67f88b1ab979c21857dc5cd279b81b47876) - [view on LGTM.com](https://lgtm.com/projects/g/dockstore/dockstore-cli/rev/pr-6f3d80ce3df1722bd06b1750606b030bab80e0b4)
> 
> **new alerts:**
> 
>     * 2 for Dereferenced variable may be null

@wshands @Richard-Hansen Just checking that you're seeing these LTGM warnings. Didn't look too closely... Is it legit? I could be wrong, but I don't recall seeing it in develop PR. Just want to make sure there is not a potential bug here.

**Comment by Richard-Hansen** [comment]
> > This pull request **introduces 2 alerts** when merging [f557469](https://github.com/dockstore/dockstore-cli/commit/f5574693e9b70eee90414a80e14e1739c1ca4cb8) into [b3dfa67](https://github.com/dockstore/dockstore-cli/commit/b3dfa67f88b1ab979c21857dc5cd279b81b47876) - [view on LGTM.com](https://lgtm.com/projects/g/dockstore/dockstore-cli/rev/pr-6f3d80ce3df1722bd06b1750606b030bab80e0b4)
> > **new alerts:**
> > ```
> > * 2 for Dereferenced variable may be null
> > ```
> 
> @wshands @Richard-Hansen Just checking that you're seeing these LTGM warnings. Didn't look too closely... Is it legit? I could be wrong, but I don't recall seeing it in develop PR. Just want to make sure there is not a potential bug here.

This is expected. When the publish/unpublish command is processed, we don't explicitly check for a null return from the tool/workflow API call, as it would throw an error if the entry is not found.

I can add an assertion for the modifications I'm already making to this area to cover the null check warning.

**Review by coverbeck — APPROVED** [review]

**Pull request merged**